net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -1022,7 +1022,7 @@ mwan3_set_user_iptables_rule()
 				     ${dest_port:+-m} ${dest_port:+multiport} ${dest_port:+--dports} "$dest_port" \
 				     -m mark --mark 0/$MMX_MASK \
 				     -m comment --comment "$1" \
-				     -j LOG --log-level "$loglevel" --log-prefix "MWAN3($1)" &> /dev/null
+				     -j LOG --log-level "$loglevel" --log-prefix "MWAN3($1)"
```

Comment:
I would leave it like this for now and drop this commit, we first have to think about how to do the logging. As you have already noted, there are many places in the code that simply put the sterr to /dev/null, which is not nice.